INSTALL.md

```diff
@@ -69,40 +69,29 @@ brew info elasticsearch
 ```
 
 ### MYSQL Database
-Ensure your database is running.
+Start MYSQL with the following command:
```

Comment:
I'll keep it for now, but perhaps `setup.sh` can be moved further up to replace this section later on.